distribution/docs/src/main/jdocs/content/_security-framework/security-filtering-contents.adoc

```diff
@@ -0,0 +1,64 @@
+:title: Filtering
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:children: na
+:order: 07
+:summary: Intro to security filtering.
+
+Metacard filtering is performed in a Access plugin that occurs after a query has been performed.
```

Comment:
"Metacard filtering is performed in a Access plugin that occurs after a query has been performed." ---> "Metacard filtering is performed by a Access plugin. The plugin is a post-query plugin that is run query has been performed, but before the results are returned to the requestor."

Reply:
How about `Metacard filtering is performed by the <<_filter_plugin,Filter Plugin>> after a query has been performed, but before the results are returned to the requestor.`

(The link will be broken until [DDF-2890](https://github.com/codice/ddf/pull/2275) is merged.)